app-crypt/snow/snow-20130616.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="Hides information in text files using trailing whitespace"
+DESCRIPTION="SNOW (Steganographic Nature Of Whitespace) is a steganography \
+tool which conceals messages in ASCII text by appending spaces to the end of \
+lines. Has an option to encrypt the message first to further secure it."
+HOMEPAGE="http://www.darkside.com.au/snow/"
+COPYRIGHT="Matthew Kwan"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="http://www.darkside.com.au/snow/$portVersionedName.tar.gz"
```

Comment:
Although
```
SOURCE_URI="http://www.darkside.com.au/snow/$portVersionedName.tar.gz"
```
is perfectly OK, we prefer to avoid $portVersionedName because when building for a secondary architecture it contains a _x86 os _x86_gcc2 suffix.
So even if this recipe does not have support for secondary architectures I would still recommend to use:
```
SOURCE_URI="http://www.darkside.com.au/snow/snow-$portVersion.tar.gz"
```